msal/src/main/java/com/microsoft/identity/client/internal/api/BrokerClientIdRefreshTokenAccessor.java

```diff
@@ -62,15 +62,15 @@
     public static @Nullable
     String get(@NonNull final Context context,
                @NonNull final String accountObjectId) throws MsalClientException {
-        final String methodName = "getBrokerRefreshToken";
+        final String methodTag = TAG + ":getBrokerRefreshToken";
```

Comment:
Should this simply be kept as ":get", for consistency?

Reply:
Or can we change the actual method name to getBrokerRefreshToken?

Reply:
That's right. Thanks for this.